packages/pools/src/interface.ts

```diff
@@ -2,7 +2,12 @@ import { Dec, Int } from "@keplr-wallet/unit";
 
 /** Interface for base pool data and basic operations on that data. */
 export interface BasePool {
-  get type(): "weighted" | "stable" | "concentrated" | "transmuter";
+  get type():
+    | "weighted"
+    | "stable"
+    | "concentrated"
+    | "transmuter"
+    | "astroport-pcl";
```

Comment:
It appears your branch is quite out of date. I updated this to be it's own type, can you please merge stage into your branch? Thanks.